samples/cs/Getting Started/index.html

```diff
@@ -0,0 +1,209 @@
+<!DOCTYPE html>
+<html>
+    
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>ZAČÍNÁME S BRACKETS</title>
+        <meta name="description" content="Interaktivní průvodce začátečníka v Brackets.">
+        <link rel="stylesheet" href="main.css">
+    </head>
+    <body>
+        
+        <h1>ZAČÍNÁME S BRACKETS</h1>
+        <h2>Tohle je váš průvodce!</h2>
+        
+        <!--
+            VYTVOŘENO S <3 A S JAVASCRIPTEM
+        -->
+        
+        <p>
+            Vítejte v Brackets, moderním open-source editoru kódu, který rozumí webdesignu. Odlehčeném,
+            a přesto výkonném editoru kódu, který prolíná textový editor s vizuálními nástroji, takže dostanete
+            správné množství pomoci, kdy budete chtít.
+        </p>
+        
+        <!--
+            CO JE BRACKETS?
+        -->
+        <p>
+            <em>Brackets je jiný druh editoru.</em>
+            Brackets obsahuje některé unikátní prvky, jako rychlou úpravu, živý náhled a další, které v jiných
+            editorech pravděpodobně nenajdete. Navíc je Brackets napsáno v JavaScriptu, HTML a CSS. To znamená,
```

Comment:
There is little typo. Correct version is `Navíc je Brackets napsán v JavaScriptu, HTML a CSS.` Just drop the letter `o`.